webapp/lib/model.dart

```diff
@@ -142,15 +165,24 @@ enum TagStyle {
   Important,
 }
 
-Configuration changeCommsPackage = new Configuration()
-  ..availableTags = tags
-  ..hasAllTags = {}
-  ..containsLastInTurnTags = {'denial': TagStyle.Normal , 'rumour': TagStyle.Normal}
-  ..hasNoneTags = {'escalate': TagStyle.Normal, 'STOP': TagStyle.Normal}
-  ..suggestedReplies = messages
-  ..addsTags = {'Organic conversation appreciation': TagStyle.Normal, 'Organic conversation hostility': TagStyle.Normal,
-    'RP Substance appreciation': TagStyle.Normal, 'RP Substance hostility': TagStyle.Normal};
-
+Map<String, Configuration> packageConfigurationData = {
+  'Change Communications': new Configuration()
+    ..availableTags = tags
+    ..hasAllTags = {}
+    ..containsLastInTurnTags = {'denial': TagStyle.Normal , 'rumour': TagStyle.Normal}
+    ..hasNoneTags = {'escalate': TagStyle.Normal, 'STOP': TagStyle.Normal}
+    ..suggestedReplies = changeCommunicationsSuggestedReplies
+    ..addsTags = {'Organic conversation appreciation': TagStyle.Normal, 'Organic conversation hostility': TagStyle.Normal,
+      'RP Substance appreciation': TagStyle.Normal, 'RP Substance hostility': TagStyle.Normal},
+    'Urgent conversations': new Configuration()
+      ..availableTags = tags
+      ..hasAllTags = {'escalate': TagStyle.Important}
+      ..containsLastInTurnTags = {}
+      ..hasNoneTags = {}
+      ..suggestedReplies = escalatesSuggestedReplies
+      ..addsTags = {'de-escalate': TagStyle.Normal, 'no-escalate': TagStyle.Normal},
+      'Open conversations': null
```

Comment:
Open conversation could be just an empty `new Configuration()`.

I'd actually also suggest that we initialise all of the attributes of the `Configuration` class with empty maps & lists rather than null, so in this map we can skip setting attributes to empty maps.

Reply:
Agree. Added